_pages/packaging-resources.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: splash
```

Comment:
this is actually the new page that reads in the stubs and content above. so this feels like a lot of new pages but really all of them appear as cards on this single new page!